### ISSUE TITLE
Improve LDAP/AD/OIDC doc

### ISSUE THREAD

**Issue by hillaryfraley** [sensu/sensu-docs#2307]
### Description
We should improve our LDAP/AD/OIDC documentation to make it easier for customers to understand and apply. This topic is complex and implementation can be difficult. We should improve our LDAP/AD/OIDC doc so that customers can be successful at implementing these auth options. 

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [x] update (add missing or refresh existing content)
- [x] improvement (improve docs content or format)

### Affected pages
https://docs.sensu.io/sensu-go/latest/installation/auth/

### Expected behavior
Customers can follow our LDAP/AD/OIDC auth instructions and successfully implement the auth option they choose. The docs include helpful examples and cover troubleshooting for all errors customers are likely to encounter.

### Current behavior
Our LDAP/AD/OIDC docs could be clearer and easier to follow. We may have some errors or unclear distinctions in our docs.

### Corrections needed
- Add to documentation of possible errors for LDAP and introduce documentation for possible errors for AD (improve https://docs.sensu.io/sensu-go/latest/installation/auth/#ldap-troubleshooting at minimum)
- Resolve `person` vs `user` (explain the distinction, clarify why we use one or the other in our examples, and provide an alternative example)

### Additional context
- Discussion in #reliability slack: https://sensu.slack.com/archives/C3MHLUP46/p1585153213018200
- Person vs. user: https://stackoverflow.com/questions/53459810/difference-between-people-and-users-ou-in-active-directory

**Comment by calebhailey** [comment]
Is this possibly related, or should I create a separate issue for this? https://github.com/sensu/web/issues/334#issuecomment-685133003 

Looks like this user needed to configure an additional setting to get OKTA OIDC working.

**Comment by hillaryfraley** [comment]
@calebhailey I think we may have addressed [this part of the problem](https://github.com/sensu/web/issues/334#issuecomment-684029389) in https://github.com/sensu/sensu-docs/pull/2702.

But is there a second problem with what the "Initiate Login URI" should be? https://github.com/sensu/web/issues/334#issuecomment-685133003

**Comment by hillaryfraley** [comment]
Reopening -- closed the wrong issue with #2816

**Comment by hillaryfraley** [comment]
Next step: re-test our Okta documentation against 6.2 and see if we can reproduce the problem described in https://github.com/sensu/web/issues/334#issuecomment-685133003. CR will try to test during office hours.

**Comment by hillaryfraley** [comment]
CR planning Asana card here: https://app.asana.com/0/1201099774497750/1201412849409886/f

**Comment by hillaryfraley** [comment]
I created a new Okta developer account and a separate account for a 30-day free trial. I do not see the classic UI option in Okta anymore -- the steps in our docs are specific to the classic UI. An Okta [support doc](https://support.okta.com/help/s/question/0D50Z00008G7Vd6/make-classic-ui-default?language=en_US) indicates that Okta turns off the dev console in favor of classic UI only at user request.

I think it makes sense for us to rewrite our instructions to follow the Okta dev console steps (I clicked around through creating a new app with the dev console and I think I was able to find everything).

This leaves two questions:
1. Do our steps still work? Before testing this, I suggest rewriting them to follow the dev console UI.
2. Should our steps include setting the `Initiate Login URI` (or at least mention an example of when it might be necessary)? Given [this user's comment](https://github.com/sensu/web/issues/334#issuecomment-684951801), it seems like setting the `Initiate Login URI` is not always necessary.
